code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -19,6 +19,8 @@
 	can_be_held = TRUE
 	pet_bonus = TRUE
 	pet_bonus_emote = "woofs happily!"
+	ai_controller = /datum/ai_controller/dog
+	stop_automated_movement = TRUE
```

Comment:
You need to remove all the old AI behavior too.

Reply:
Yeah, that's covered in my checklist of stuff to do